inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -376,6 +376,45 @@ void FrontEnd::parseLayer(const Model& model, const ie::CNNLayerPtr& layer, cons
     }
 }
 
+void FrontEnd::processTrivialCases(const Model& model) {
+    DataVector trivialInputs, trivialOutputs;
+
+    for (const auto& data : model->datas()) {
+        if (data->usage() == DataUsage::Input || data->usage() == DataUsage::Intermediate || data->usage() == DataUsage::Const) {
+            if (data->numConsumers() == 0 && data->childDataToShapeEdges().size() == 0) {
+                trivialInputs.push_back(data);
+            }
+        }
+        if (data->usage() == DataUsage::Output || data->usage() == DataUsage::Intermediate) {
```

Comment:
Similar question: is it okay to have intermediate data object with no producer?

Reply:
Answered above.